src/index.js

```diff
@@ -1,4 +1,39 @@
 import './styles.css';
-import submitFunc from './submitscores.js';
+import addScore from './submitscores.js';
+// reference to markup elements
+const refresh = document.getElementById('refresh');
+const submitButton = document.getElementById('submit');
+const recentList = document.getElementById('recent-list');
+const yourName = document.getElementById('your-name');
+const yourScore = document.getElementById('your-score');
 
-submitFunc();
\ No newline at end of file
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/24c0c3c116974ac49488d4eb0267ade3/scores';
+const showScore = async () => {
+  recentList.innerHTML = '';
+
+  await fetch(url)
+    .then((response) => response.json())
+    .then((json) => {
+      const scores = json.result;
+
+      scores.forEach((item) => {
+        const list = document.createElement('li');
+        list.innerHTML = `
+      ${item.user} :    ${item.score}`;
+        recentList.appendChild(list);
+      });
+    });
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225498659-d053aa02-efe5-4a12-a3b7-aead3b688154.png)

**[Optional]**

- [ ] When i click on the button then it duplicates the existing data in your application which you get from the api 🤔 Kindly fix this issue 👍